gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/membership/model/Membership.java

```diff
@@ -31,7 +31,10 @@ public class Membership {
     private ReferenceType referenceType;
     private String referenceId;
     private String roleId;
-    private String source;
+
+    @Builder.Default
+    private String source = "system";
```

Comment:
can we extract "system" string as a constant or an enum with value somewhere ?

Reply:
From what I saw the source is always `system` I didn't push the investigation I had enough of membership for a while...